VMEncryption/main/CommandExecutor.py

```diff
@@ -74,8 +74,8 @@ def timeout_process():
 
         if int(return_code) != 0:
             msg = "Command {0} failed with return code {1}".format(command_to_execute, return_code)
-            msg += "\nstdout:\n" + stdout
-            msg += "\nstderr:\n" + stderr
+            msg += "\nstdout:\n" + stdout.decode("utf-8")
```

Comment:
updating as follows:  from builtins import str, using str() instead of decode for p2/p3 compat, when process communicator in use, also changing that to use str() representation